### DISCUSSION TITLE
Eliminating Render Blocking CSS and measuring page rendering time with Performance API | Leo's dev blog – stories, insights, and ideas

### DISCUSSION THREAD

**Discussion by giscus** [hta218/leohuynh.dev#68] [Announcements]
# Eliminating Render Blocking CSS and measuring page rendering time with Performance API | Leo's dev blog – stories, insights, and ideas

Learn how to improve your website's performance and enhance user experience by avoiding render blocking CSS. Using the Chrome Performance API to measure and optimize page rendering time.

https://www.leohuynh.dev/blog/render-blocking-css-and-chrome-performance-api

**Comment by hta218** [comment]
#### Hey, thanks for checking out this post! 💡

If you found it useful (or have some ideas for improvement), I’d love to hear from you! Leave a comment below with your thoughts, or give a reaction to show your support. It means a lot to me!

Cheers 🍺
